Library/Homebrew/brew.sh

```diff
@@ -51,9 +51,13 @@ HOMEBREW_CACHE="${HOMEBREW_CACHE:-${HOMEBREW_DEFAULT_CACHE}}"
 HOMEBREW_LOGS="${HOMEBREW_LOGS:-${HOMEBREW_DEFAULT_LOGS}}"
 HOMEBREW_TEMP="${HOMEBREW_TEMP:-${HOMEBREW_DEFAULT_TEMP}}"
 
-# These referenced variables are set by bin/brew
 # Don't need to handle a default case.
-# shellcheck disable=SC2154,SC2249
+# HOMEBREW_LIBRARY set by bin/brew
+# shellcheck disable=SC2249,SC2154
+# shellcheck source=Homebrew/cmd/shellenv.sh
+# shellcheck source=Homebrew/cmd/formulae.sh
+# shellcheck source=Homebrew/cmd/casks.sh
+# shellcheck source=Homebrew/prefix.sh
```

Comment:
These don't seem to be necessary. I've removed them all and `shellcheck` still passes as long as the `--source-path` is passed (which `brew style` does). Any starting with `source=Homebrew/` (which seems to be all of them) can just be deleted.

Reply:
@MikeMcQuaid If I remove the above directives, `shellcheck` does not pass for me and gives the following output.  (could be differences in our shell/shellcheck versions) My shellcheck's version is 0.7.1 
```

In /usr/local/Homebrew/Library/Homebrew/brew.sh line 62:
  shellenv)            source "${HOMEBREW_LIBRARY}/Homebrew/cmd/shellenv.sh"; homebrew-shellenv; exit 0 ;;
                              ^-- SC1090: Can't follow non-constant source. Use a directive to specify location.


In /usr/local/Homebrew/Library/Homebrew/brew.sh line 63:
  formulae)            source "${HOMEBREW_LIBRARY}/Homebrew/cmd/formulae.sh"; homebrew-formulae; exit 0 ;;
                              ^-- SC1090: Can't follow non-constant source. Use a directive to specify location.


In /usr/local/Homebrew/Library/Homebrew/brew.sh line 64:
  casks)               source "${HOMEBREW_LIBRARY}/Homebrew/cmd/casks.sh"; homebrew-casks; exit 0 ;;
                              ^-- SC1090: Can't follow non-constant source. Use a directive to specify location.


In /usr/local/Homebrew/Library/Homebrew/brew.sh line 66:
  --prefix*)           source "${HOMEBREW_LIBRARY}/Homebrew/prefix.sh"; homebrew-prefix "$@" && exit 0 ;;
                              ^-- SC1090: Can't follow non-constant source. Use a directive to specify location.

For more information:
  https://www.shellcheck.net/wiki/SC1090 -- Can't follow non-constant source....```

Reply:
@GauthamGoli I'm on 0.7.2. Try to `brew upgrade shellcheck` and use `brew style` rather than `shellcheck` and see if that works? If not: please just remove anyway and as long as it passes CI we're good 👍🏻

Reply:
Updated!